### ISSUE TITLE
Failed to authenticate!

### ISSUE THREAD

**Issue by e33et** [jixxed/ed-odyssey-materials-helper#215]
Everytime I open Elite Dangerous Odyssey Materials Helper Launcher, I've got this issue/error that I need to re-authenticate Frontier API, how can I get rid of it ? I have checked in settings, but can't find the authentication button!

<img width="1516" alt="Odyssey_matrial_helper_error" src="https://github.com/jixxed/ed-odyssey-materials-helper/assets/66057180/b698b988-cbce-4c5e-96e4-df6ca92c3cf0">

Best regards
/e33et

**Comment by jixxed** [comment]
You can find it under Settings to authenticate. Frontier resets your authentication every month.

**Comment by e33et** [comment]
Yep find it, would be nice if you could add a minimize to system tray option, otherwise it works really good!

**Comment by jixxed** [comment]
this is now possible
